specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json

```diff
@@ -2495,6 +2495,18 @@
             "archiveNameFormat": {
               "type": "string",
               "description": "Blob naming convention for archive, e.g. {Namespace}/{EventHub}/{PartitionId}/{Year}/{Month}/{Day}/{Hour}/{Minute}/{Second}. Here all the parameters (Namespace,EventHub .. etc) are mandatory irrespective of order"
+            },
+            "dataLakeSubscriptionId": {
+              "type": "string",
```

Comment:
it is stil a breaking change . only readonly optional (new) fields are permitted in existing api version
https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo

Reply:
apologies for late reply. 

I guess previously we had added new optional properties.

Reply:
what does not providing this value on the second PUT mean? Will the service ignore it?
What is the behavior if this property is not provided on the 1st PUT? How does the customer go back to this behavior once this value is provided?

Reply:
@v-Ajnava - ping. please ping me once you have answered above and we can close on it.

Reply:
@ravbhatnagar I discussed with Arun and yes this will be a breaking change. we will take it new API version. 

I will close the PR